adobe/react-spectrum
# Add main as a wrapper for story content

Closes 
https://github.com/adobe/react-spectrum/issues/1268
This seems to be a real quick fix -- simply adding a `main` wrapper in `.storybook/custom-addons/provider/index.js` would do the trick.

Before:
![Screen Shot 2022-03-28 at 2 55 32 PM](https://user-images.githubusercontent.com/19467235/160495661-bbe4dd7b-e35a-4e5f-93f6-06e2965eed77.png)

After:
![Screen Shot 2022-03-28 at 2 56 34 PM](https://user-images.githubusercontent.com/19467235/160495677-12b94a1c-1ef9-4e85-ab9b-dede23124a62.png)


## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. add the aXe devtools to your chrome extension
2. choose a story arbitrarily and open it in a new tab
3. run aXe against the entire page, check if the warning is still there

## 🧢 Your Project:
